public/app/features/dashboard-scene/sharing/ShareButton/share-externally/ShareExternally.test.tsx

```diff
@@ -116,7 +116,14 @@ async function buildAndRenderScenario({
 
   render(<drawer.Component model={drawer} />);
 
-  await waitForElementToBeRemoved(screen.getByText('Loading configuration'));
+  const loadingElement = screen.queryByText('Loading configuration');
+  if (loadingElement) {
+    await waitForElementToBeRemoved(loadingElement);
+  }
+
+  await waitFor(() => {
+    expect(screen.getByTestId(shareExternallySelector.container)).toBeInTheDocument();
+  });
```

Comment:
You can remove the `if` and `waitFor` logic and instead do this:

`expect(await screen.findByTestId(shareExternallySelector.container)).toBeInTheDocument();`

Reply:
This change has been reverted!